broker/core/src/main/resources/kapua-broker-setting.properties

```diff
@@ -5,7 +5,6 @@ broker.ip_resolver_class_name=
 # Broker IP used by the default resolver.
 broker.ip=
 broker.id_resolver_class_name=
-broker.jaxb_context_class_name=org.eclipse.kapua.broker.core.BrokerJAXBContextProvider
```

Comment:
JAXB is not initialized centrally within the DI context startup - configuration and customization of the jaxb context should be done through appropriate DI context manipulation (nominally: proper module configuration)